tests/gocase/unit/type/json/json_test.go

```diff
@@ -152,4 +152,36 @@ func TestJson(t *testing.T) {
 		require.NoError(t, err)
 	})
 
+	t.Run("JSON.ARRLEN basics", func(t *testing.T) {
+		require.NoError(t, rdb.Do(ctx, "JSON.SET", "a", "$", `{"a1":[1,2],"a2":[[1,5,7],[8],[9]],"i":1,"s":"string","o":{"a3":[1,1,1]}}`).Err())
```

Comment:
You also can add the command to delete the key (`a`) at the beginning of the test to prevent any naming clashes now or in the future.